### PR TITLE
DBZ-4354 Fix incorrect command for copying connector plug-in files

### PR DESCRIPTION
**Pull request by roldanbob**
[DBZ-4354](https://issues.redhat.com/browse/DBZ-4354)

This change corrects errors in the legacy instructions for creating the Dockerfile for deploying a connector, and also includes some minor related language and formatting edits. The change affects all of the connectors that are included in the RHI product and is confined to the content that is conditionalized for downstream use. The changes are not rendered in the community version of the documentation. 

Note that the change has implications for the downstream-only Getting Started and OCP installation guides, which are maintained in a separate repo. I'll address those changes separately.

I'd like to merge this change in before completing the work on DBZ-3991 to incorporate the instructions based on the updated Strimzi/Streams deployment mechanism.

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -2366,24 +2366,23 @@ Apply this CR to the same OpenShift instance where you applied the `KafkaConnect
...
+.. Create a `Dockerfile` that uses `{DockerKafkaConnect}` as the base image.
+For example, from a terminal window, enter the following command:
 +
 [source,shell,subs="+attributes,+quotes"]
 ----
 cat <<EOF >debezium-container-for-postgresql.yaml // <1>
 FROM {DockerKafkaConnect}
 USER root:root
-COPY ./_<my-plugins>_/ /opt/kafka/plugins/ // <2>
+COPY ./debezium-connector-postgres/ /opt/kafka/plugins/debezium-connector-postgres
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2377]: Ok, so there's a discrepancy between what's reported in the Jira issue (based on what's shown in rendered downstream docs) and the original version in this file here. Because this actually was correct then:

```
COPY ./_<my-plugins>_/ /opt/kafka/plugins/
```

It'd copy an entire plug-ins directory to `plugins` in the image. Whereas what was reported in Jira is wrong, as it copies only a single connector's contents to `plugins`:

```
COPY ./debezium-connector-postgres/ /opt/kafka/plugins/ 
```

So I am wondering where this discrepancy is coming from. In fact, what we have shown here (before this proposed change) arguably is the better approach, as it implicitly shows how to add multiple connectors. All in all, my preference would be to leave this as is (sans the minor formatting changes you did) and rather make whatever we have downstream consistent with that too (ideally, as you know, this would be sourced from these upstream files to begin with, exactly to avoid this kind of inconsistency).

**Review by gunnarmorling — CHANGES_REQUESTED** [review]
Hum, hum, one comment inline. I think this should be addressed slightly differently.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc (hunk 1 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc:2377]: Sorry, @gunnarmorling, but I'm confused now. Are you saying that I should revert the change and go back to:

> `COPY ./_<my-plugins>_/ /opt/kafka/plugins/ `?

Or, actually, to incorporate my suggested formatting change:

>   `COPY ./my-plugins/ /opt/kafka/plugins/ `

If we do that, of course I'll add back in the sentence about replacing `my-plugins` with the actual directory name.

Regarding inconsistencies between upstream and downstream, I'm not sure what you're referring to. In the downstream doc, the lines in question are sourced directly from the upstream, but they are conditionalized specifically for downstream use.

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc:2377]: I have no strong opinion on the formatting; but the COPY command should not contain the specific connector directory, but it should copy the entire `<my-plugins>` directory. I.e. what we had already was correct.

> Regarding inconsistencies between upstream and downstream, I'm not sure what you're referring to.

I was referring to the issue report in Jira. There it suggests the current downstream docs would contain this:

```
COPY ./debezium-connector-postgres/ /opt/kafka/plugins/ 
```

As we've agreed above, that's wrong, it should copy the `<my-plugins>` directory. But now when looking at the rendered downstream docs linked from the issue, I can't actually find that wrong form. So you may double check with the reporter (it's a Red Hatter) where he actually saw that wrong form or whether he mixed up things.

Long story short: what we had was and is correct in terms of contents, feel free to apply any formatting/style changes as you see fit.

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc:2377]: >  As we've agreed above, that's wrong, it should copy the <my-plugins> directory. But now when looking at the rendered downstream docs linked from the issue, I can't actually find that wrong form. So you may double check with the reporter (it's a Red Hatter) where he actually saw that wrong form or whether he mixed up things.

> Long story short: what we had was and is correct in terms of contents,

@gunnarmorling 
Given that, I'm inclined to close this PR. Earlier today I reached out to the issue reporter and asked for input, but I haven't yet received a response.

**Comment by gunnarmorling** [comment]
Ok, closing this for now then. Thx, @roldanbob!
